bridges/ethMultiversX/bridgeExecutor_test.go

```diff
@@ -1655,3 +1666,16 @@ func TestBridgeExecutor_ValidateBatch(t *testing.T) {
 	assert.True(t, result)
 	assert.True(t, validateBatchCalled)
 }
+
+// CompareBytes compares two byte slices and returns true if they are equal.
+func CompareBytes(a, b []byte) bool {
```

Comment:
`bytes.Equal(b1, b2)` instead of this? or simply `string(b1) == string(b2)` (which bytes.Equal also does)

Reply:
removed